usaspending_api/etl/rapidloader.py

```diff
@@ -107,5 +108,6 @@ def complete_process(self) -> None:
             take_snapshot(self.elasticsearch_client, self.config["index_name"], settings.ES_REPOSITORY)
 
         if self.config["is_incremental_load"]:
+            toggle_refresh_on(self.elasticsearch_client, self.config["index_name"])
```

Comment:
The refresh will remain off. 
I thought about putting in some kind of try-finally, but it may just be something we need to administer in handling errors. 
The effect will only be that ad-hoc deletes/update to the data won't automatically refresh. We shouldn't be doing those types of ad-hoc things normally, and when we do, we'll need to double check that the changed data was refreshed. 

The normal course of things is that if an incremental load fails, we fix the problem, and re-run it til it works, at which point the refresh will be set correctly again at the end. 
That or, we nuke and pave a botched index, at the end of which the refresh will be correct also. 

So while not air-tight, I think it's a reasonable concession.